tools/pylib/boutdata/data.py

```diff
@@ -599,11 +607,47 @@ def keys(self):
         """
         return self.varNames
 
-    def evolvingVariables(self):
-        """Return a list of names of time-evolving variables
 
+    @property
+    def shiftAngle(self):
+        """
+        The shift angle, calculated from zShift the first time it is used
+        """
+        if self._shiftAngle is None:
+            try:
+                nz = self["MZ"]
+            except KeyError:
+                try:
+                    nz = self.attributes("mz")
+                except KeyError:
+                    nz = self.attributes["mesh:nz"]
+            zlength = self["dz"]*nz
+            self._shiftAngle = 2.*numpy.pi*self["zShift"]/zlength
+        return self._shiftAngle
+
+    def getShifted(self, name):
+        """
+        Return a variable shifted by zShift: if BOUT++ output is in
+        'unshifted', i.e. toroidal, coordinates, this function returns a field
+        in field-aligned coordinates
```

Comment:
This sounds a bit like it checks if the output file is in unshifted space, and always does the "correct" thing, but if I understand correctly, it unconditionally shifts the variable?

Always doing the "correct thing" would be best; if that's not possible currently, please could you change the docstring to say no checking is done (or something along those lines)